app/controllers/filler/skiplagged.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+const pokeRadar = require(__appbase + 'services/skiplagged');
+
+module.exports = {
+    /*
+     * for inserting pokemon data to MongoDB
+     */
+    insertToDb: function () {
+        pokeRadar.search();
+    }
+};
```

Comment:
This whole file looks the same as pokeradar, so just a single file can be used.
To handlethe workflow for listen,
in config.js, we could maintain a map of collection-name vs listener.
Like pokeRadar and skipLagged could map to a common listener. And in listen.js, instead of accessing the listener by collection name, it can be fetched from this map.

Thiss way, we can avoid duplication of code.